app/src/main/java/com/example/recipeopedia/fragments/ProfileFragment.java

```diff
@@ -0,0 +1,59 @@
+package com.example.recipeopedia.fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.recipeopedia.R;
+import com.parse.ParseUser;
+
+public class ProfileFragment extends Fragment {
+    private static final String TAG = "ProfileFragment";
+    private ImageView ivProfileImage;
+    private TextView tvUsername;
+    private TextView tvBio;
+    private Button btnEditProfile;
+
+    public ProfileFragment() {}
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_profile, container, false);
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        ivProfileImage = view.findViewById(R.id.ivProfileImage);
+        tvUsername = view.findViewById(R.id.tvUsername);
+        tvBio = view.findViewById(R.id.tvBio);
+        ivProfileImage = view.findViewById(R.id.ivProfileImage);
+
+        btnEditProfile = view.findViewById(R.id.btnEditProfile);
+        btnEditProfile.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.i(TAG, "onClick edit button");
+                // go to edit profile screen
```

Comment:
I'm assuming this is still a work in progress?

Reply:
Yep, it should be updated in my next commit!